### PR TITLE
Temporarily remove uglifier to help debug white screen of death in production

### PR DESCRIPTION
**Pull request by jaredhirsch**
@meandavejustice Mind taking a look?

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -88,9 +88,6 @@ gulp.task('scripts', shouldLint('js-lint', 'lint'), function scriptsTask() {
     .pipe(source('app.js'))
     .pipe(buffer())
     .pipe(gulpif(IS_DEBUG, sourcemaps.init({loadMaps: true})))
```
Comment (meandavejustice) [inline gulpfile.js:90]: do we want to enable sourcemaps in prod to help debug?

**Comment by jaredhirsch** [comment]
Comment from @meandavejustice via IRC:

```
10:53 AM <JSON_voorhees> do we want to enable sourcemaps in prod?
10:53 AM <JSON_voorhees> to help debug
```

I'll make that change then bump this PR again

**Comment by jaredhirsch** [comment]
Looks like I need to fix the tests, too

**Comment by lmorchard** [comment]
Yeah, the test that failed is linting that caught a module no longer in use because of the deleted line. Might be easier to just force [`IS_DEBUG=true` here](https://github.com/6a68/testpilot/blob/e5ff5077e2e54b80ef8d4696696956f03350457a/gulpfile.js#L27), for now.

**Comment by jaredhirsch** [comment]
gah, I'm running out of time and won't be able to get this wrapped up. Pinged dave about this earlier in IRC, but forgot to close this PR. sorry :beers:

**Pull request closed without merging**